app/connectors/CustomsDataStoreConnector.scala

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package connectors
+
+import scala.concurrent.{ExecutionContext, Future}
+
+import config.AppConfig
+import javax.inject.Inject
+import models.EmailAddress
+import play.api.http.Status.NOT_FOUND
+import uk.gov.hmrc.http.{HeaderCarrier, HttpClient, Upstream4xxResponse}
+import uk.gov.hmrc.http.HttpReads.Implicits._
+
+class CustomsDataStoreConnector @Inject()(http: HttpClient)(implicit appConfig: AppConfig, ec: ExecutionContext) {
+
+  import CustomsDataStoreConnector._
+
+  def getEmailAddress(eori: String)(implicit hc: HeaderCarrier): Future[Option[EmailAddress]] =
+    http.GET[EmailAddress](verifiedEmailUrl(eori)).map(Some(_)).recover {
+      case Upstream4xxResponse(_, NOT_FOUND, _, _) => None
+    }
+}
+
+object CustomsDataStoreConnector {
+
+  def verifiedEmailPath(eori: String)(implicit appConfig: AppConfig): String =
+    s"/${appConfig.customsDataStoreContext}/eori/$eori/verified-email"
```

Comment:
Could you make `/eori/$eori/verified-email` configurable? In all other services we put them into `application.conf` so that it is easy to change if we need to. And sometimes we do for different environments.

Reply:
I can of course, np, but honestly I do not see any advantage to have a **parametric** path in application.conf because you end up having two places to verify/take care of in case of modifications (you cannot just change application.conf ignoring to check how this change will affect the code).
I understand the team is using this practice in other service too, but I think my comment still is valid.
WDYT (the team) @MaciejRewera @glidester @caoilte

Reply:
I'm not strongly in favor of one style or the other tbh, but at this point in project maturity I'd go for consistency of approach (rule of least surprise!)

Reply:
:+1: